platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -29,9 +30,25 @@ NS_ASSUME_NONNULL_BEGIN
  
  @param mapView The map view whose viewpoint will change.
  @param animated Whether the change will cause an animated effect on the map.
+
+ @note If `mapView:regionWillChangeAnimated:reason` is implemented this method will not be called.
  */
 - (void)mapView:(MGLMapView *)mapView regionWillChangeAnimated:(BOOL)animated;
 
+/**
+ Tells the delegate that the viewpoint depicted by the map view is about to change.
+
+ This method is called whenever the currently displayed map camera will start
+ changing for any reason.
+
+ @param mapView The map view whose viewpoint will change.
+ @param animated Whether the change will cause an animated effect on the map.
+ @param reason The reason for the camera change.
+
+ @note If this method is implemented `mapView:regionWillChangeAnimated:` will not be called.
```

Comment:
Add `-` at the beginning of the selector name so that jazzy can detect that it’s a method reference and automatically link it to the relevant documentation.